tests/storage/concurrent_vm_boot/conftest.py

```
@@ -0,0 +1,115 @@
+@pytest.fixture(scope="module")
+def started_vms_with_five_disks(created_vms_with_five_disks):
+    errors = run_vms_parallel(
+        vms=created_vms_with_five_disks,
+        func=lambda vm: vm.start(wait=False),
+        label="Failed to start VM",
+    )
+    if errors:
+        raise ClusterSanityError(
+            err_str=f"{len(errors)}/{len(created_vms_with_five_disks)} VMs failed to start: {errors}"
+        )
+    yield created_vms_with_five_disks
```

Ruff (0.16.2)

[warning] No teardown in fixture `started_vms_with_five_disks`, use `return` instead of `yield`

Replace `yield` with `return`

(PT022)